tests/foreman/api/test_contentview.py

```diff
@@ -396,11 +350,10 @@ def test_positive_publish_1(self):
         @Feature: ContentView
 
         """
-        content_view = entities.ContentView()
-        content_view.id = content_view.create_json()['id']
+        content_view = entities.ContentView().create()
         for _ in range(REPEAT):
             content_view.publish()
-        self.assertEqual(len(content_view.read_json()['versions']), REPEAT)
+        self.assertEqual(len(content_view.read().version), REPEAT)
```

Comment:
We should think better on this subject. With the new additions related to handling pluralized words maybe we should consider making more user friendly.

It is more natural to think versions being a a list than version.